doc/cascadia/Keybindings-spec.md

```diff
@@ -30,7 +30,7 @@ When the UX frontend is created, it should instantiate a `IKeyBindings` object w
 
 When it's creating its platform-dependent terminal component, it can pass the `IKeyBindings` object to that component. The component will then be able to pass that object to the terminal instance.
 
-When the terminal component calls `ITerminalInput.SendKeyEvent(uint vkey, KeyModifiers modifiers)`, the terminal will use `IKeyBindings.TryKeyChord` to see if there are any bound actions to that input. If there are, the `IKeyBindings` implementation will either handle the event by interacting with the `ITerminalInput`, or it'll invoke an event that's been registered by the frontend

```

Comment:
This change I'd probably revert - it's just a spec, and I think it'd be better to leave in it's original state, to display what we were thinking when the spec was written.

Reply:
I've reverted this change now. 🙂